docs/pages/box.js

```diff
@@ -419,26 +419,32 @@ function Example() {
             )}
           </CombinationNew>
         </MainSection.Subsection>
-        <MainSection.Subsection title="Colors">
+        <MainSection.Subsection
+          title="Colors"
+          description={`
+          The following colors can be used to change the background of Box.
+
+          ⚠️ Note that the previous options ('red', 'white', 'lightGray', 'gray', 'darkGray', 'green', 'pine', 'olive', 'blue', 'navy', 'midnight', 'purple', 'orchid', 'eggplant', 'maroon', 'watermelon', 'orange') are still valid but will be deprecated soon.
```

Comment:
What do you think about adding a section helping with this transition? A table or something that shows "prevColor --> newColor"

Reply:
We don't have this mapping yet (part of why I've split this into 2 PRs), but I'll definitely do that for the next PR to help show which are deprecated vs which have new color mappings

Reply:
nit: if it's easy, can we display "inverse" on a dark background?